gitclone/checkout_method_pr_auto_diff.go

```diff
@@ -58,13 +55,9 @@ func (c checkoutPRDiffFile) do(gitCmd git.Git, fetchOptions fetchOptions, fallba
 		log.Warnf("Could not apply patch (%s): %v", c.patchFile, err)
 		log.Warnf("Falling back to manual merge...")
 
-		return fallbackToManualMergeOnApplyError(
-			gitCmd,
-			c.params.PRManualMergeParam,
-			c.params.ForkPRManualMergeParam,
-			fetchOptions,
-			fallback,
-		)
+		if err := c.params.PRManualMergeStrategy.do(gitCmd, fetchOptions, fallback); err != nil {
+			return fmt.Errorf("Applying patch (%s) failed: %v", c.patchFile, err)
```

Comment:
`Manual merge failed: %s, err`

Reply:
I believe if the manual merge fails for some reason, it will indicate its failure (with something like this: `Manual merge failed`). My intention here was to signal that the fallback option has also failed, so there is no further way in order to apply the patch. 

I agree that the error message in question does not clearly reflect what I just mentioned here, so if you agree, I would just update this error message to express it better.